README

```diff
@@ -53,7 +53,7 @@ Do the usual thing:
     make
     make install
 
-You must have the dependencies installed in a location configure will find,
```

Comment:
I think "will" is better here, and a better fix would be "in a location *where* configure will find _them_,"

Reply:
I see, your suggestion makes sense (it's the "dependencies", not the "location", that `configure` will find). I will update